targets/lantiq-xrx200

```diff
@@ -32,7 +32,14 @@ device('avm-fritz-box-3370-rev-2-micron-nand', 'avm_fritz3370-rev2-micron', {
 
 device('avm-fritz-box-7360-sl', 'avm_fritz7360sl', {
 	factory = false,
-	aliases = {'avm-fritz-box-7360-v1', 'avm-fritz-box-7360-v2'},
+	aliases = {'avm-fritz-box-7360-v1'},
+})
+
+device('avm-fritz-box-7360-v2', 'avm_fritz7360-v2', {
+	factory = false,
+	manifest_aliases = {
+		'avm,fritz7360sl' -- upgrade from OpenWrt 22.03
```

Comment:
The alias is definitely incorrect, `image_name()` will never return a string with a comma in it.

Reply:
Thanks for spotting